torch/testing/_internal/common_methods_invocations.py

```diff
@@ -370,6 +372,55 @@ def sample_inputs(self, device, dtype, requires_grad=False):
             ]
 
 
+class HermitianOpInfo(OpInfo):
+    """Operator information for Hermitian functions
+    These are functions that take Hermitian matrices as input.
+    They require a modified function to be tested for gradcheck.
```

Comment:
Elaborate on the reason they need a modified function.

Reply:
The finite-difference algorithm for calculating derivatives does not preserve the Hermitian property of the input and returns incorrect results failing the gradcheck.